pyproject.toml

```diff
@@ -29,6 +29,8 @@ requires = ["setuptools",
             "wheel"]
 build-backend = "setuptools.build_meta"
 
+[tool.setuptools_scm]
```

Comment:
this block is empty- does it really need to be here?

Reply:
yep, an empty block is fine if we dont want to add any extra settings